tools/prepare_python_system_tests.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+import json
+import os
+import subprocess
+import tempfile
+import time
+import urllib.request
+import zipfile
+
+parser = argparse.ArgumentParser(description='Downloads the latest release nimi-python and runs system tests on the specified driver.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('-d', '--driver', required=True, type=str, help='Driver Name.')
+parser.add_argument('-pv', '--python-version', required=False, type=str, help='Python version to be run.', default=None)
+parser.add_argument('-pb', '--python-bitness', required=False, type=str, help='Python bitness to be run.', default=None)
```

Comment:
Have examples so syntax is obvious. Is bitness "32" or "32-bit"?  Is version "3.8" or "38" ?

Reply:
Changed the output to look like this:
```
usage: prepare_python_system_tests.py [-h] -d {nidcpower,nidigital,nidmm,nifake,nifgen,nimodinst,niscope,nise,niswitch,nisync,nitclk}
                                      [-pv {py35,py36,py37,py38}]
                                      [-pb {None,--32}]

Downloads the latest release artifacts from nimi-python and runs system tests
on the specified python package.

optional arguments:
  -h, --help            show this help message and exit
  -d {nidcpower,nidigital,nidmm,nifake,nifgen,nimodinst,niscope,nise,niswitch,nisync,nitclk}, --driver {nidcpower,nidigital,nidmm,nifake,nifgen,nimodinst,niscope,nise,niswitch,nisync,nitclk}
                        Python package name. (default: None)
  -pv {py35,py36,py37,py38}, --python-version {py35,py36,py37,py38}
                        Python version to be run. This is used to invoke the
                        appropriate tox environment. (default: py38)
  -pb {None,--32}, --python-bitness {None,--32}
                        Python bitness to be run. This is passed to tox.
                        (default: None)
```